### PR TITLE
Add page info to autogenerated pages

### PR DESCRIPTION
**Pull request by sftim**
Tell the reader when they're viewing an autogenerated page, in case they want to report an issue - it should help make reports about autogenerated content stand out. [[example](https://deploy-preview-30267--kubernetes-io-main-staging.netlify.app/docs/reference/kubernetes-api/service-resources/)]

### CODE REVIEW

File: data/i18n/en/en.toml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # i18n strings for the English (main) site.
 # NOTE: Please keep the entries in alphabetical order when editing
+[auto_generated_edit_notice]
+other = "(autogenerated page)"
+
+[auto_generated_pageinfo]
+other = """<p>This page is automatically generated.</p><p>If you find something wrong with this page and plan to report it, please mention that you're reporting an issue with a page that is autogenerated. The fix may need to happen elsewhere in the Kubernetes project.</p>"""
```
Comment (kbhawkey) [inline data/i18n/en/en.toml:7]: Hi @sftim. I like the idea of adding a mark or brief statement that the page is generated.
I am not sure if it makes sense to include more content on the page about logging an issue (I think it detracts from the page content).
I think a brief message could be placed after the title of the page?

```
This page is automatically generated.
```
or at the bottom of the page without the border and extra text about reporting an issue,
```
This page is automatically generated by the [API Reference Generator Tool] (provide link to tool repo).
```

Q: The link ((auto-generated page)) in the right hand navigation is missing the icon? I think that link item is somewhat confusing, especially with the parenthesis.

Comment (sftim) [reply data/i18n/en/en.toml:7]: I matched what we're already doing for third party content.

If I changed the styling of the pageinfo and simplify the message to “This page is automatically generated.”, does that cover what you'd like to see?

We don't have information about which generator tool was used, so we can't add the kind of link you had in mind (without extra work).

The missing icon looks as I expect here; similar to the advice we give for pages that contain 3rd party content.

Comment (kbhawkey) [reply data/i18n/en/en.toml:7]: OK. Some more ideas: The "(auto-generated page)" links to the message at the end of the page. Have you tried to add a partial before the main block (like the theme's "deprecation-warning")? That way, you would not need a link in the right hand toc. The partial could write a brief statement and link to the `Contribute` section of the docs (and explain how to make upstream changes in k8s and possibly link to the generator tool source).

Comment (tengqm) [reply data/i18n/en/en.toml:7]: I have the same suggestion. A simple and clear message at the very beginning of the page would be desirable. I don't see the value of generating a link to a message that appears at the bottom of the page.

**Comment by sftim** [comment]
I'll close this for now. Might pick it up in the future.
/close

**Pull request closed without merging**